tuf/client/updater.py

```diff
@@ -1495,14 +1496,21 @@ def _get_metadata_file(self, metadata_role, remote_filename,
           metadata_spec_version = metadata_signable['signed']['spec_version']
           metadata_spec_major_version = int(metadata_spec_version.split('.')[0])
           code_spec_major_version = int(tuf.SPECIFICATION_VERSION.split('.')[0])
-
+          
           if metadata_spec_major_version != code_spec_major_version:
             raise tuf.exceptions.UnsupportedSpecificationError(
                 'Downloaded metadata that specifies an unsupported '
                 'spec_version.  This code supports major version number: ' +
                 repr(code_spec_major_version) + '; however, the obtained '
                 'metadata lists version number: ' + str(metadata_spec_version))
 
+          #report to user if minor versions do not match, continue with update
+          if pkg_resources.parse_version(metadata_spec_version) != pkg_resources.parse_version(tuf.SPECIFICATION_VERSION):
+            logger.info("Downloaded metadata that specifies a higher minor " +
+                "spec_version. This code has version " + str(tuf.SPECIFICATION_VERSION) +
+                "and the metadata lists version number " + str(metadata_spec_version) +
+                ". The update will continue as the major versions match.")
+
```

Comment:
Please try to [wrap lines at character 79](https://github.com/secure-systems-lab/code-style-guidelines#maximum-line-length).

Reply:
OT: This is actually something we should have the linter check as well.